doc/classes/File.xml

```diff
@@ -226,7 +227,7 @@
 			<argument index="2" name="compression_mode" type="int" default="0">
 			</argument>
 			<description>
-				Open a compressed file for reading or writing. The compression_mode can be set as one of the COMPRESSION_* constants.
+				Opens a compressed file for reading or writing. The compression_mode can be set as one of the COMPRESSION_* constants.
```

Comment:
Second sentence:

Use COMPRESSION_* constants to set [code]compression_mode[/code].